scripts/nn/layers/tanh.dml

```diff
@@ -29,21 +29,13 @@ forward = function(matrix[double] X)
   /*
    * Computes the forward pass for a tanh nonlinearity layer.
    *
-   *   ```
-   *   tanh(x) = (e^x - e^-x) / (e^x + e^-x)
-   *           = 2 * sigmoid(2x) - 1
-   *   ```
-   *
```

Comment:
I would leave this comment in, as it gives the math for a tanh function.

Reply:
Done.